config/webpack/environment.js

```diff
@@ -1,3 +1,10 @@
 const { environment } = require('@rails/webpacker')
 
+const webpack = require('webpack')
+environment.plugins.append('Provide', new webpack.ProvidePlugin({
+  $: 'jquery',
+  jQuery: 'jquery',
+  Popper: ['popper.js', 'default']
+}))
```

Comment:
What does this do?

Reply:
This will point Rails towards Webpacker and your new dependencies.
https://blog.makersacademy.com/how-to-install-bootstrap-and-jquery-on-rails-6-da6e810c1b87

Reply:
I tried removing this and the bootstrap still appears to be working so I'm gonna remove it.

Reply:
turns out I needed this locally... so I added it back in with 2ca0be7